<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>670</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);
</string>
  </property>
  <widget class="QPushButton" name="botao_add_camisa_branca_simpsonss">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>280</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nike 200204</family>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);
border-style:outset;
border-width: 2px;
border-radius: 15px;
border-color:black;
}
QPushButton:pressed{
color:black;
background-color:white;
border-style:outset;
border-width: 2px;
border-radius: 15px;
border-color:black;
}</string>
   </property>
   <property name="text">
    <string>ADICIONAR AO CARRINHO</string>
   </property>
  </widget>
  <widget class="QPushButton" name="botao_add_camisa_preta_onep">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>560</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nike 200204</family>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);
border-style:outset;
border-width: 2px;
border-radius: 15px;
border-color:black;
}
QPushButton:pressed{
color:black;
background-color:white;
border-style:outset;
border-width: 2px;
border-radius: 15px;
border-color:black;
}</string>
   </property>
   <property name="text">
    <string>ADICIONAR AO CARRINHO</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>230</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>8</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;CAMISA NARUTO - BRANCA&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;R$ 70,00&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>30</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nike 200204</family>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;CAMISAS&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_26">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>360</y>
     <width>161</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/produtos/Camisas/One Piece, Naruto e Simpsons/onepiece-preta.png);
background-color: rgb(243, 243, 243);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/icons/icones/Gamepad (3).png);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>80</y>
     <width>181</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/produtos/Camisas/One Piece, Naruto e Simpsons/naturo-branca.png);
background-color: rgb(243, 243, 243);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="botao_add_camisa_branca_onep">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>280</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nike 200204</family>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);
border-style:outset;
border-width: 2px;
border-radius: 15px;
border-color:black;
}
QPushButton:pressed{
color:black;
background-color:white;
border-style:outset;
border-width: 2px;
border-radius: 15px;
border-color:black;
}</string>
   </property>
   <property name="text">
    <string>ADICIONAR AO CARRINHO</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>80</y>
     <width>151</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/produtos/Camisas/One Piece, Naruto e Simpsons/simpsons-branco.png);
background-color: rgb(243, 243, 243);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="botao_add_camisa_preta_naruto">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>560</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nike 200204</family>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);
border-style:outset;
border-width: 2px;
border-radius: 15px;
border-color:black;
}
QPushButton:pressed{
color:black;
background-color:white;
border-style:outset;
border-width: 2px;
border-radius: 15px;
border-color:black;
}</string>
   </property>
   <property name="text">
    <string>ADICIONAR AO CARRINHO</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>230</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>8</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;CAMISA ONE PIECE - BRANCA&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;R$ 70,00&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="botao_voltar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>630</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);
border-style:outset;
border-width: 2px;
border-radius: 15px;
border-color:black;
}
QPushButton:pressed{
color:black;
background-color:white;
border-style:outset;
border-width: 2px;
border-radius: 15px;
border-color:black;
}</string>
   </property>
   <property name="text">
    <string>VOLTAR</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_25">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>510</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>8</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;CAMISA SIMPSONS - PRETA&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;R$ 70,00&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_23">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>360</y>
     <width>151</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/produtos/Camisas/One Piece, Naruto e Simpsons/simpsons-preto.png);
background-color: rgb(243, 243, 243);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="botao_add_camisa_branca_simpsons">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>230</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>8</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;CAMISA SIMPSONS - BRANCA&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;R$ 70,00&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="botao_add_camisa_branca_naruto">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>280</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nike 200204</family>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);
border-style:outset;
border-width: 2px;
border-radius: 15px;
border-color:black;
}
QPushButton:pressed{
color:black;
background-color:white;
border-style:outset;
border-width: 2px;
border-radius: 15px;
border-color:black;
}</string>
   </property>
   <property name="text">
    <string>ADICIONAR AO CARRINHO</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>80</y>
     <width>161</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/produtos/Camisas/One Piece, Naruto e Simpsons/onepiece-branca.png);
background-color: rgb(243, 243, 243);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_24">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>510</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>8</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;CAMISA ONE PIECE - PRETA&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;R$ 70,00&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="botao_carrinho">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>10</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/icons/icones/carr.png);
image: url(:/imgicons/icones/carr.png);

color: rgb(255, 255, 255);
border-style:outset;
border-width: 2px;
border-color:black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="botao_add_camisa_preta_simpsons">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>560</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nike 200204</family>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);
border-style:outset;
border-width: 2px;
border-radius: 15px;
border-color:black;
}
QPushButton:pressed{
color:black;
background-color:white;
border-style:outset;
border-width: 2px;
border-radius: 15px;
border-color:black;
}</string>
   </property>
   <property name="text">
    <string>ADICIONAR AO CARRINHO</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_22">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>510</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>8</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;CAMISA NARUTO - PRETA&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;R$ 70,00&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_27">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>360</y>
     <width>181</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/produtos/Camisas/One Piece, Naruto e Simpsons/naturo-preta.png);
background-color: rgb(243, 243, 243);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QComboBox" name="tamanho_camisa_branca_simpsons">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>250</y>
     <width>41</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka Display Semibold</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(173, 173, 173);
border-bottom-color: rgb(173, 173, 173);</string>
   </property>
   <item>
    <property name="text">
     <string>P</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>M</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>G</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="tamanho_camisa_preto_simpsons">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>530</y>
     <width>41</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka Display Semibold</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(173, 173, 173);
border-bottom-color: rgb(173, 173, 173);</string>
   </property>
   <item>
    <property name="text">
     <string>P</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>M</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>G</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="tamanho_camisa_branca_onepiece">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>250</y>
     <width>41</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka Display Semibold</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(173, 173, 173);
border-bottom-color: rgb(173, 173, 173);</string>
   </property>
   <item>
    <property name="text">
     <string>P</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>M</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>G</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="tamanho_camisa_preto_onepiece">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>530</y>
     <width>41</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka Display Semibold</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(173, 173, 173);
border-bottom-color: rgb(173, 173, 173);</string>
   </property>
   <item>
    <property name="text">
     <string>P</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>M</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>G</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="tamanho_camisa_branca_naruto">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>250</y>
     <width>41</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka Display Semibold</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(173, 173, 173);
border-bottom-color: rgb(173, 173, 173);</string>
   </property>
   <item>
    <property name="text">
     <string>P</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>M</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>G</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="tamanho_camisa_preto_naruto">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>530</y>
     <width>41</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka Display Semibold</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(173, 173, 173);
border-bottom-color: rgb(173, 173, 173);</string>
   </property>
   <item>
    <property name="text">
     <string>P</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>M</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>G</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources>
  <include location="../produtos/produtos.qrc"/>
  <include location="img-icons.qrc"/>
 </resources>
 <connections/>
</ui>
